autotest/gcore/tiff_write.py

```diff
@@ -7331,6 +7331,42 @@ def test_tiff_write_jpeg_incompatible_of_paletted():
 
 
 ###############################################################################
+# Test blocksize overriding while creating (internal) overviews
+# on a newly created dataset
+
+def test_tiff_write_186_blocksize():
+    md = gdaltest.tiff_drv.GetMetadata()
+    if md['DMD_CREATIONOPTIONLIST'].find('WEBP') == -1:
+        pytest.skip()
+
+    blockSizes = [128,256]
```

Comment:
wouldn't testing just 64 for example be enough ? (64 is never a default block size)

Reply:
now testing for 64 and 256, in case one day one of those becomes the default :)